tabs/src/translations/zh-CN.json

```diff
@@ -1,15 +1,70 @@
 {
     "translation": {
+
+        "Query Runner": {
+            "Key": "(zh-cn) Key",
+            "Value": "(zh-cn) Value",
+            "Run query": "(zh-cn) RUN QUERY",
+            "Request body": "(zh-cn) Request Body",
+            "Request headers": "(zh-cn) Request Headers",
+            "Response body": "(zh-cn) Response Body",
+            "Response headers": "(zh-cn) Response Headers",
+            "Response": "(zh-cn) Response",
+            "Warning": "(zh-cn) To use Graph requests, you must connect the app to a resource in order to use resource-specific consent queries. Learn more here.",
+            "Add": "(zh-cn) ADD"
+        },
+
+        "Connected Resources": {
+            "Teams Channel ID": "(zh-cn) Teams Channel ID",
+            "Chat": "(zh-cn) Chat",
+            "Chat ID": "(zh-cn) Chat ID",
+            "No Connected Resources": "(zh-cn) No connected resources. Please connect to a Teams or Chats instance."
```

Comment:
Suggested: "You must connect to an instance of a team or a chat to make this request."